[2025-08-19T09:25:18.507+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-19T09:25:18.665+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:31.039964+00:00 [queued]>
[2025-08-19T09:25:18.732+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:31.039964+00:00 [queued]>
[2025-08-19T09:25:18.733+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 4
[2025-08-19T09:25:18.869+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): transform_golden> on 2025-08-19 09:11:31.039964+00:00
[2025-08-19T09:25:18.908+0000] {standard_task_runner.py:63} INFO - Started process 2392 to run task
[2025-08-19T09:25:18.917+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'transform_golden', 'manual__2025-08-19T09:11:31.039964+00:00', '--job-id', '606', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpa7e56r9u']
[2025-08-19T09:25:18.935+0000] {standard_task_runner.py:91} INFO - Job 606: Subtask transform_golden
[2025-08-19T09:25:19.567+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:31.039964+00:00 [running]> on host 54bf3a3eaeab
[2025-08-19T09:25:20.914+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='transform_golden' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T09:11:31.039964+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T09:11:31.039964+00:00'
[2025-08-19T09:25:20.923+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-19T09:25:21.214+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-19T09:25:21.232+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'docker exec spark-master spark-submit --master spark://spark-master:7077 --deploy-mode client --packages org.apache.spark:spark-avro_2.12:3.5.1 /opt/***/jobs/transformation/transform_to_golden_dataset.py']
[2025-08-19T09:25:21.339+0000] {subprocess.py:86} INFO - Output:
[2025-08-19T09:25:36.505+0000] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/opt/bitnami/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-08-19T09:25:36.766+0000] {subprocess.py:93} INFO - Ivy Default Cache set to: /opt/bitnami/spark/.ivy2/cache
[2025-08-19T09:25:36.769+0000] {subprocess.py:93} INFO - The jars for the packages stored in: /opt/bitnami/spark/.ivy2/jars
[2025-08-19T09:25:36.796+0000] {subprocess.py:93} INFO - org.apache.spark#spark-avro_2.12 added as a dependency
[2025-08-19T09:25:36.801+0000] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-c4706f68-eabc-4457-9f52-5870200762b2;1.0
[2025-08-19T09:25:36.803+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-08-19T09:25:37.416+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-avro_2.12;3.5.1 in central
[2025-08-19T09:25:37.603+0000] {subprocess.py:93} INFO - 	found org.tukaani#xz;1.9 in central
[2025-08-19T09:25:37.663+0000] {subprocess.py:93} INFO - :: resolution report :: resolve 849ms :: artifacts dl 14ms
[2025-08-19T09:25:37.666+0000] {subprocess.py:93} INFO - 	:: modules in use:
[2025-08-19T09:25:37.668+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-avro_2.12;3.5.1 from central in [default]
[2025-08-19T09:25:37.668+0000] {subprocess.py:93} INFO - 	org.tukaani#xz;1.9 from central in [default]
[2025-08-19T09:25:37.669+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:25:37.670+0000] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2025-08-19T09:25:37.671+0000] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-08-19T09:25:37.672+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:25:37.673+0000] {subprocess.py:93} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-08-19T09:25:37.674+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:25:37.676+0000] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-c4706f68-eabc-4457-9f52-5870200762b2
[2025-08-19T09:25:37.678+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-08-19T09:25:37.700+0000] {subprocess.py:93} INFO - 	0 artifacts copied, 2 already retrieved (0kB/22ms)
[2025-08-19T09:25:38.677+0000] {subprocess.py:93} INFO - 25/08/19 09:25:38 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-08-19T09:25:46.229+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SparkContext: Running Spark version 3.5.1
[2025-08-19T09:25:46.230+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SparkContext: OS info Linux, 6.11.0-29-generic, amd64
[2025-08-19T09:25:46.231+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SparkContext: Java version 17.0.12
[2025-08-19T09:25:46.286+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceUtils: ==============================================================
[2025-08-19T09:25:46.287+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-08-19T09:25:46.288+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceUtils: ==============================================================
[2025-08-19T09:25:46.291+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SparkContext: Submitted application: Processed Zone - Golden Dataset
[2025-08-19T09:25:46.349+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-08-19T09:25:46.364+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceProfile: Limiting resource is cpu
[2025-08-19T09:25:46.366+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-08-19T09:25:46.717+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SecurityManager: Changing view acls to: spark
[2025-08-19T09:25:46.753+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SecurityManager: Changing modify acls to: spark
[2025-08-19T09:25:46.754+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SecurityManager: Changing view acls groups to:
[2025-08-19T09:25:46.767+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SecurityManager: Changing modify acls groups to:
[2025-08-19T09:25:46.798+0000] {subprocess.py:93} INFO - 25/08/19 09:25:46 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: spark; groups with view permissions: EMPTY; users with modify permissions: spark; groups with modify permissions: EMPTY
[2025-08-19T09:25:52.531+0000] {subprocess.py:93} INFO - 25/08/19 09:25:52 INFO Utils: Successfully started service 'sparkDriver' on port 46247.
[2025-08-19T09:25:52.760+0000] {subprocess.py:93} INFO - 25/08/19 09:25:52 INFO SparkEnv: Registering MapOutputTracker
[2025-08-19T09:25:52.968+0000] {subprocess.py:93} INFO - 25/08/19 09:25:52 INFO SparkEnv: Registering BlockManagerMaster
[2025-08-19T09:25:53.132+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-08-19T09:25:53.134+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-08-19T09:25:53.165+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-08-19T09:25:53.409+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-0ec6df7e-44fd-4664-86d4-926ba7dbd48c
[2025-08-19T09:25:53.458+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-08-19T09:25:53.543+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-08-19T09:25:53.944+0000] {subprocess.py:93} INFO - 25/08/19 09:25:53 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-08-19T09:25:54.287+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2025-08-19T09:25:54.320+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2025-08-19T09:25:54.327+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2025-08-19T09:25:54.423+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO Utils: Successfully started service 'SparkUI' on port 4043.
[2025-08-19T09:25:54.859+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar at spark://b208641feb52:46247/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar with timestamp 1755595546203
[2025-08-19T09:25:54.862+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar at spark://b208641feb52:46247/jars/org.tukaani_xz-1.9.jar with timestamp 1755595546203
[2025-08-19T09:25:54.873+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar at spark://b208641feb52:46247/files/org.apache.spark_spark-avro_2.12-3.5.1.jar with timestamp 1755595546203
[2025-08-19T09:25:54.875+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar to /tmp/spark-671c8c7d-e1dc-45f7-8fd5-5da643f4bf8e/userFiles-a5cd1170-c319-407b-9c4d-a676c44c54c0/org.apache.spark_spark-avro_2.12-3.5.1.jar
[2025-08-19T09:25:54.975+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar at spark://b208641feb52:46247/files/org.tukaani_xz-1.9.jar with timestamp 1755595546203
[2025-08-19T09:25:54.977+0000] {subprocess.py:93} INFO - 25/08/19 09:25:54 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar to /tmp/spark-671c8c7d-e1dc-45f7-8fd5-5da643f4bf8e/userFiles-a5cd1170-c319-407b-9c4d-a676c44c54c0/org.tukaani_xz-1.9.jar
[2025-08-19T09:25:56.410+0000] {subprocess.py:93} INFO - 25/08/19 09:25:56 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2025-08-19T09:25:58.116+0000] {subprocess.py:93} INFO - 25/08/19 09:25:58 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.4:7077 after 1388 ms (0 ms spent in bootstraps)
[2025-08-19T09:26:00.056+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20250819092600-0012
[2025-08-19T09:26:00.090+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 40505.
[2025-08-19T09:26:00.098+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO NettyBlockTransferService: Server created on b208641feb52:40505
[2025-08-19T09:26:00.119+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-08-19T09:26:00.192+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, b208641feb52, 40505, None)
[2025-08-19T09:26:00.223+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO BlockManagerMasterEndpoint: Registering block manager b208641feb52:40505 with 434.4 MiB RAM, BlockManagerId(driver, b208641feb52, 40505, None)
[2025-08-19T09:26:00.244+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, b208641feb52, 40505, None)
[2025-08-19T09:26:00.256+0000] {subprocess.py:93} INFO - 25/08/19 09:26:00 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, b208641feb52, 40505, None)
[2025-08-19T09:26:01.347+0000] {subprocess.py:93} INFO - 25/08/19 09:26:01 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-08-19T09:26:02.525+0000] {subprocess.py:93} INFO - 25/08/19 09:26:02 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2025-08-19T09:26:02.529+0000] {subprocess.py:93} INFO - 25/08/19 09:26:02 INFO SharedState: Warehouse path is 'file:/opt/bitnami/spark/spark-warehouse'.
[2025-08-19T09:26:11.149+0000] {subprocess.py:93} INFO - 25/08/19 09:26:11 INFO InMemoryFileIndex: It took 485 ms to list leaf files for 1 paths.
[2025-08-19T09:26:17.330+0000] {subprocess.py:93} INFO - 25/08/19 09:26:17 INFO InMemoryFileIndex: It took 162 ms to list leaf files for 1 paths.
[2025-08-19T09:26:29.390+0000] {subprocess.py:93} INFO - 25/08/19 09:26:29 INFO InMemoryFileIndex: It took 128 ms to list leaf files for 1 paths.
[2025-08-19T09:26:29.688+0000] {subprocess.py:93} INFO - 25/08/19 09:26:29 INFO InMemoryFileIndex: It took 92 ms to list leaf files for 1 paths.
[2025-08-19T09:26:34.643+0000] {subprocess.py:93} INFO - 25/08/19 09:26:34 INFO InMemoryFileIndex: It took 32 ms to list leaf files for 1 paths.
[2025-08-19T09:26:34.740+0000] {subprocess.py:93} INFO - 25/08/19 09:26:34 INFO InMemoryFileIndex: It took 40 ms to list leaf files for 1 paths.
[2025-08-19T09:26:36.576+0000] {subprocess.py:93} INFO - 25/08/19 09:26:36 INFO InMemoryFileIndex: It took 21 ms to list leaf files for 1 paths.
[2025-08-19T09:26:36.813+0000] {subprocess.py:93} INFO - 25/08/19 09:26:36 INFO InMemoryFileIndex: It took 44 ms to list leaf files for 1 paths.
[2025-08-19T09:26:42.650+0000] {subprocess.py:93} INFO - 25/08/19 09:26:42 INFO InMemoryFileIndex: It took 90 ms to list leaf files for 1 paths.
[2025-08-19T09:26:43.061+0000] {subprocess.py:93} INFO - 25/08/19 09:26:43 INFO InMemoryFileIndex: It took 108 ms to list leaf files for 1 paths.
[2025-08-19T09:26:45.198+0000] {subprocess.py:93} INFO - 25/08/19 09:26:45 INFO InMemoryFileIndex: It took 22 ms to list leaf files for 1 paths.
[2025-08-19T09:26:45.501+0000] {subprocess.py:93} INFO - 25/08/19 09:26:45 INFO InMemoryFileIndex: It took 31 ms to list leaf files for 1 paths.
[2025-08-19T09:26:48.075+0000] {subprocess.py:93} INFO - 25/08/19 09:26:48 INFO InMemoryFileIndex: It took 97 ms to list leaf files for 1 paths.
[2025-08-19T09:26:48.384+0000] {subprocess.py:93} INFO - 25/08/19 09:26:48 INFO InMemoryFileIndex: It took 45 ms to list leaf files for 1 paths.
[2025-08-19T09:26:51.773+0000] {subprocess.py:93} INFO - 25/08/19 09:26:51 INFO InMemoryFileIndex: It took 97 ms to list leaf files for 1 paths.
[2025-08-19T09:26:52.285+0000] {subprocess.py:93} INFO - 25/08/19 09:26:52 INFO InMemoryFileIndex: It took 64 ms to list leaf files for 1 paths.
[2025-08-19T09:26:53.726+0000] {subprocess.py:93} INFO - 25/08/19 09:26:53 INFO InMemoryFileIndex: It took 50 ms to list leaf files for 1 paths.
[2025-08-19T09:26:53.946+0000] {subprocess.py:93} INFO - 25/08/19 09:26:53 INFO InMemoryFileIndex: It took 24 ms to list leaf files for 1 paths.
[2025-08-19T09:26:55.343+0000] {subprocess.py:93} INFO - 25/08/19 09:26:55 INFO InMemoryFileIndex: It took 80 ms to list leaf files for 1 paths.
[2025-08-19T09:26:55.947+0000] {subprocess.py:93} INFO - 25/08/19 09:26:55 INFO InMemoryFileIndex: It took 158 ms to list leaf files for 1 paths.
[2025-08-19T09:27:09.859+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:09.866+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#7),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None) END),(cast(views#7 as bigint) > 0)
[2025-08-19T09:27:09.890+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:09.892+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#32)
[2025-08-19T09:27:09.922+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:09.924+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#500),isnotnull(CASE WHEN (length(trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None) END),(cast(views#500 as bigint) > 0)
[2025-08-19T09:27:09.928+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:09.936+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#525)
[2025-08-19T09:27:09.971+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:09.971+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#993),isnotnull(CASE WHEN (length(trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None) END),(cast(views#993 as bigint) > 0)
[2025-08-19T09:27:09.975+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:09.976+0000] {subprocess.py:93} INFO - 25/08/19 09:27:09 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#1018)
[2025-08-19T09:27:10.000+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.003+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#1486),isnotnull(CASE WHEN (length(trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None) END),(cast(views#1486 as bigint) > 0)
[2025-08-19T09:27:10.009+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.011+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#1511)
[2025-08-19T09:27:10.054+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.056+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#1979),isnotnull(CASE WHEN (length(trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None) END),(cast(views#1979 as bigint) > 0)
[2025-08-19T09:27:10.060+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.062+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2004)
[2025-08-19T09:27:10.096+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.100+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#2472),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None) END),(cast(views#2472 as bigint) > 0)
[2025-08-19T09:27:10.101+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.102+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2497)
[2025-08-19T09:27:10.146+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.147+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#2965),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None) END),(cast(views#2965 as bigint) > 0)
[2025-08-19T09:27:10.147+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.148+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2990)
[2025-08-19T09:27:10.181+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.183+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#3458),isnotnull(CASE WHEN (length(trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None) END),(cast(views#3458 as bigint) > 0)
[2025-08-19T09:27:10.187+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.188+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#3483)
[2025-08-19T09:27:10.210+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.211+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#3951),isnotnull(CASE WHEN (length(trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None) END),(cast(views#3951 as bigint) > 0)
[2025-08-19T09:27:10.217+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.218+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#3976)
[2025-08-19T09:27:10.240+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:27:10.240+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#4444),isnotnull(CASE WHEN (length(trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None) END),(cast(views#4444 as bigint) > 0)
[2025-08-19T09:27:10.240+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:27:10.241+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#4469)
[2025-08-19T09:27:10.997+0000] {subprocess.py:93} INFO - 25/08/19 09:27:10 WARN SparkStringUtils: Truncated the string representation of a plan since it was too large. This behavior can be adjusted by setting 'spark.sql.debug.maxToStringFields'.
[2025-08-19T09:27:14.229+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1644.71418 ms
[2025-08-19T09:27:14.231+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1611.211877 ms
[2025-08-19T09:27:14.380+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1801.871902 ms
[2025-08-19T09:27:14.435+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1868.298918 ms
[2025-08-19T09:27:14.437+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1816.424021 ms
[2025-08-19T09:27:14.502+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1880.160505 ms
[2025-08-19T09:27:14.545+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1947.295665 ms
[2025-08-19T09:27:14.547+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1947.132214 ms
[2025-08-19T09:27:14.556+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1978.946323 ms
[2025-08-19T09:27:14.573+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO CodeGenerator: Code generated in 1984.121369 ms
[2025-08-19T09:27:14.644+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_5 stored as values in memory (estimated size 200.9 KiB, free 434.2 MiB)
[2025-08-19T09:27:14.657+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_3 stored as values in memory (estimated size 200.9 KiB, free 434.0 MiB)
[2025-08-19T09:27:14.657+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_0 stored as values in memory (estimated size 200.9 KiB, free 433.8 MiB)
[2025-08-19T09:27:14.738+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_1 stored as values in memory (estimated size 200.9 KiB, free 433.4 MiB)
[2025-08-19T09:27:14.739+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_6 stored as values in memory (estimated size 200.9 KiB, free 433.2 MiB)
[2025-08-19T09:27:14.749+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_4 stored as values in memory (estimated size 200.9 KiB, free 432.8 MiB)
[2025-08-19T09:27:14.753+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_8 stored as values in memory (estimated size 200.9 KiB, free 433.0 MiB)
[2025-08-19T09:27:14.754+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_7 stored as values in memory (estimated size 200.9 KiB, free 432.8 MiB)
[2025-08-19T09:27:14.782+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_9 stored as values in memory (estimated size 200.9 KiB, free 432.6 MiB)
[2025-08-19T09:27:14.810+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 INFO MemoryStore: Block broadcast_2 stored as values in memory (estimated size 200.9 KiB, free 432.4 MiB)
[2025-08-19T09:27:15.770+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_7_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.775+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_6_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.790+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_3_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.799+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_8_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.816+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_9_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.824+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_0_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.830+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_1_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.863+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_5_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.865+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO BlockManagerInfo: Added broadcast_7_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.4 MiB)
[2025-08-19T09:27:15.869+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_2_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:15.870+0000] {subprocess.py:93} INFO - 25/08/19 09:27:15 INFO MemoryStore: Block broadcast_4_piece0 stored as bytes in memory (estimated size 34.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:16.009+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:27:16.015+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_5_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:27:16.022+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 7 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.049+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_3_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:27:16.062+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 1 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.063+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_6_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:27:16.064+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 5 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.065+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:27:16.077+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 0 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.084+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 3 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.097+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 6 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.115+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_9_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:27:16.117+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 9 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.123+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.129+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.133+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.150+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_8_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.1 MiB)
[2025-08-19T09:27:16.150+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_2_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.1 MiB)
[2025-08-19T09:27:16.151+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO BlockManagerInfo: Added broadcast_4_piece0 in memory on b208641feb52:40505 (size: 34.7 KiB, free: 434.1 MiB)
[2025-08-19T09:27:16.165+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 8 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.168+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.174+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.174+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167907 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.217+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.217+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.218+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 4 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.235+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO SparkContext: Created broadcast 2 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:16.237+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:16.243+0000] {subprocess.py:93} INFO - 25/08/19 09:27:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:27:18.082+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.246+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.248+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.250+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.261+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.261+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.262+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.264+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.275+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.288+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 0 (MapPartitionsRDD[23] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.290+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.295+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.309+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.315+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.321+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:27:18.336+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_10 stored as values in memory (estimated size 14.1 KiB, free 432.1 MiB)
[2025-08-19T09:27:18.339+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_10_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:18.342+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_10_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.1 MiB)
[2025-08-19T09:27:18.346+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 10 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.412+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 0 (MapPartitionsRDD[23] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.415+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 0.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.472+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.473+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.473+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.473+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.474+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 1 (MapPartitionsRDD[36] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.484+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_11 stored as values in memory (estimated size 14.1 KiB, free 432.1 MiB)
[2025-08-19T09:27:18.493+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_11_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.1 MiB)
[2025-08-19T09:27:18.499+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_11_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.505+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 11 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.509+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 1 (MapPartitionsRDD[36] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.512+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 1.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.516+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.522+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.524+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.526+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.536+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 2 (MapPartitionsRDD[27] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.553+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_12 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.560+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_12_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.574+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_12_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.578+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 12 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.579+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 2 (MapPartitionsRDD[27] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.587+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 2.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.594+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.602+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.606+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.606+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.607+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 3 (MapPartitionsRDD[16] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.622+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_13 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.628+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_13_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.629+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_13_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.630+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 13 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.634+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 3 (MapPartitionsRDD[16] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.635+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 3.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.636+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.636+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.637+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.639+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.640+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 4 (MapPartitionsRDD[37] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.645+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_14 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.647+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_14_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.649+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_14_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.650+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 14 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.655+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 4 (MapPartitionsRDD[37] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.657+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 4.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.662+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.663+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.663+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.663+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.668+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 5 (MapPartitionsRDD[38] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.676+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_15 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.679+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_15_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.685+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_15_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.690+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 15 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.695+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 5 (MapPartitionsRDD[38] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.696+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 5.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.701+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.703+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.706+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.710+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.715+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 6 (MapPartitionsRDD[29] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.730+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_16 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.735+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_16_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:27:18.745+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_16_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.750+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 16 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.756+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 6 (MapPartitionsRDD[29] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.758+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 6.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.764+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.766+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.767+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.773+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.774+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 7 (MapPartitionsRDD[31] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.807+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_17 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.819+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_17_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.831+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_17_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.836+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 17 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.836+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 7 (MapPartitionsRDD[31] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.837+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 7.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.837+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.845+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.849+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.851+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.851+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 8 (MapPartitionsRDD[39] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.871+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_18 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.885+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_18_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.896+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_18_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.909+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 18 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.920+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 8 (MapPartitionsRDD[39] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.921+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 8.0 with 2 tasks resource profile 0
[2025-08-19T09:27:18.930+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Got job 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:27:18.932+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Final stage: ResultStage 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:27:18.935+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:27:18.938+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:27:18.939+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting ResultStage 9 (MapPartitionsRDD[25] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:27:18.949+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_19 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.952+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO MemoryStore: Block broadcast_19_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:27:18.958+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO BlockManagerInfo: Added broadcast_19_piece0 in memory on b208641feb52:40505 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:27:18.964+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO SparkContext: Created broadcast 19 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:27:18.970+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 9 (MapPartitionsRDD[25] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:27:18.972+0000] {subprocess.py:93} INFO - 25/08/19 09:27:18 INFO TaskSchedulerImpl: Adding task set 9.0 with 2 tasks resource profile 0
[2025-08-19T09:27:33.460+0000] {subprocess.py:93} INFO - 25/08/19 09:27:33 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:27:48.463+0000] {subprocess.py:93} INFO - 25/08/19 09:27:48 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:03.461+0000] {subprocess.py:93} INFO - 25/08/19 09:28:03 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:18.461+0000] {subprocess.py:93} INFO - 25/08/19 09:28:18 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:33.461+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:48.470+0000] {subprocess.py:93} INFO - 25/08/19 09:28:48 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:29:03.459+0000] {subprocess.py:93} INFO - 25/08/19 09:29:03 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:29:18.461+0000] {subprocess.py:93} INFO - 25/08/19 09:29:18 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:29:33.461+0000] {subprocess.py:93} INFO - 25/08/19 09:29:33 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:29:48.458+0000] {subprocess.py:93} INFO - 25/08/19 09:29:48 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:30:03.466+0000] {subprocess.py:93} INFO - 25/08/19 09:30:03 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:30:18.462+0000] {subprocess.py:93} INFO - 25/08/19 09:30:18 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:30:19.153+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819092600-0012/0 on worker-20250819090904-172.18.0.12-34859 (172.18.0.12:34859) with 4 core(s)
[2025-08-19T09:30:19.195+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819092600-0012/0 on hostPort 172.18.0.12:34859 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:30:19.220+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819092600-0012/1 on worker-20250819090904-172.18.0.17-46791 (172.18.0.17:46791) with 4 core(s)
[2025-08-19T09:30:19.237+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819092600-0012/1 on hostPort 172.18.0.17:46791 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:30:19.266+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819092600-0012/2 on worker-20250819090906-172.18.0.16-36117 (172.18.0.16:36117) with 4 core(s)
[2025-08-19T09:30:19.268+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819092600-0012/2 on hostPort 172.18.0.16:36117 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:30:19.583+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819092600-0012/0 is now RUNNING
[2025-08-19T09:30:19.602+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819092600-0012/1 is now RUNNING
[2025-08-19T09:30:19.680+0000] {subprocess.py:93} INFO - 25/08/19 09:30:19 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819092600-0012/2 is now RUNNING
[2025-08-19T09:30:33.464+0000] {subprocess.py:93} INFO - 25/08/19 09:30:33 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:30:42.683+0000] {subprocess.py:93} INFO - 25/08/19 09:30:42 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.12:45436) with ID 0,  ResourceProfileId 0
[2025-08-19T09:30:43.420+0000] {subprocess.py:93} INFO - 25/08/19 09:30:43 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.16:54698) with ID 2,  ResourceProfileId 0
[2025-08-19T09:30:43.752+0000] {subprocess.py:93} INFO - 25/08/19 09:30:43 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.12:37993 with 434.4 MiB RAM, BlockManagerId(0, 172.18.0.12, 37993, None)
[2025-08-19T09:30:44.002+0000] {subprocess.py:93} INFO - 25/08/19 09:30:44 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.16:37923 with 434.4 MiB RAM, BlockManagerId(2, 172.18.0.16, 37923, None)
[2025-08-19T09:30:45.680+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 0.0 in stage 0.0 (TID 0) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:45.690+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 1.0 in stage 0.0 (TID 1) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:45.704+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 0.0 in stage 1.0 (TID 2) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:45.705+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 1.0 in stage 1.0 (TID 3) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:45.851+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 0.0 in stage 3.0 (TID 4) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:45.858+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 1.0 in stage 3.0 (TID 5) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:45.860+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 0.0 in stage 2.0 (TID 6) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:45.864+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO TaskSetManager: Starting task 1.0 in stage 2.0 (TID 7) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:45.885+0000] {subprocess.py:93} INFO - 25/08/19 09:30:45 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.17:53208) with ID 1,  ResourceProfileId 0
[2025-08-19T09:30:46.506+0000] {subprocess.py:93} INFO - 25/08/19 09:30:46 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.17:33623 with 434.4 MiB RAM, BlockManagerId(1, 172.18.0.17, 33623, None)
[2025-08-19T09:30:47.936+0000] {subprocess.py:93} INFO - 25/08/19 09:30:47 INFO BlockManagerInfo: Added broadcast_12_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:48.052+0000] {subprocess.py:93} INFO - 25/08/19 09:30:47 INFO BlockManagerInfo: Added broadcast_13_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:48.053+0000] {subprocess.py:93} INFO - 25/08/19 09:30:47 INFO TaskSetManager: Starting task 0.0 in stage 4.0 (TID 8) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:48.059+0000] {subprocess.py:93} INFO - 25/08/19 09:30:48 INFO TaskSetManager: Starting task 1.0 in stage 4.0 (TID 9) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:48.123+0000] {subprocess.py:93} INFO - 25/08/19 09:30:48 INFO TaskSetManager: Starting task 0.0 in stage 5.0 (TID 10) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:30:48.124+0000] {subprocess.py:93} INFO - 25/08/19 09:30:48 INFO TaskSetManager: Starting task 1.0 in stage 5.0 (TID 11) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:30:48.228+0000] {subprocess.py:93} INFO - 25/08/19 09:30:48 INFO BlockManagerInfo: Added broadcast_10_piece0 in memory on 172.18.0.12:37993 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:48.243+0000] {subprocess.py:93} INFO - 25/08/19 09:30:48 INFO BlockManagerInfo: Added broadcast_11_piece0 in memory on 172.18.0.12:37993 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:51.231+0000] {subprocess.py:93} INFO - 25/08/19 09:30:51 INFO BlockManagerInfo: Added broadcast_14_piece0 in memory on 172.18.0.17:33623 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:51.292+0000] {subprocess.py:93} INFO - 25/08/19 09:30:51 INFO BlockManagerInfo: Added broadcast_15_piece0 in memory on 172.18.0.17:33623 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:55.010+0000] {subprocess.py:93} INFO - 25/08/19 09:30:55 INFO BlockManagerInfo: Added broadcast_5_piece0 in memory on 172.18.0.16:37923 (size: 34.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:55.066+0000] {subprocess.py:93} INFO - 25/08/19 09:30:55 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 172.18.0.16:37923 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:30:58.231+0000] {subprocess.py:93} INFO - 25/08/19 09:30:58 INFO BlockManagerInfo: Added broadcast_7_piece0 in memory on 172.18.0.12:37993 (size: 34.7 KiB, free: 434.4 MiB)
[2025-08-19T09:30:58.364+0000] {subprocess.py:93} INFO - 25/08/19 09:30:58 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 172.18.0.12:37993 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:01.198+0000] {subprocess.py:93} INFO - 25/08/19 09:31:01 INFO BlockManagerInfo: Added broadcast_2_piece0 in memory on 172.18.0.17:33623 (size: 34.7 KiB, free: 434.4 MiB)
[2025-08-19T09:31:01.261+0000] {subprocess.py:93} INFO - 25/08/19 09:31:01 INFO BlockManagerInfo: Added broadcast_9_piece0 in memory on 172.18.0.17:33623 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:03.160+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 INFO TaskSetManager: Starting task 0.0 in stage 6.0 (TID 12) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:03.286+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 INFO TaskSetManager: Starting task 1.0 in stage 6.0 (TID 13) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:03.384+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 INFO TaskSetManager: Starting task 0.0 in stage 7.0 (TID 14) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:03.647+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 WARN TaskSetManager: Lost task 1.0 in stage 2.0 (TID 7) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:03.648+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:03.649+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:03.651+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:03.652+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:03.653+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:03.654+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:03.654+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:31:03.658+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:03.658+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:03.660+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:03.662+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:03.665+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:03.668+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:03.670+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:03.670+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:03.670+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:03.675+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:03.676+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:03.678+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:03.684+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:03.687+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:03.687+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:03.688+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:03.693+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:03.693+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:03.693+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:03.697+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:03.826+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 WARN TaskSetManager: Lost task 0.0 in stage 2.0 (TID 6) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:03.827+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:03.829+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:03.830+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:03.832+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:03.833+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:03.834+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:03.837+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:31:03.838+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:03.840+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:03.841+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:03.843+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:03.844+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:03.845+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:03.846+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:03.848+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:03.849+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:03.851+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:03.852+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:03.853+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:03.855+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:03.857+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:03.858+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:03.859+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:03.861+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:03.863+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:03.864+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:03.866+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:03.867+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 INFO TaskSetManager: Starting task 0.1 in stage 2.0 (TID 15) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:03.867+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 WARN TaskSetManager: Lost task 1.0 in stage 3.0 (TID 5) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00011-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:03.868+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:03.871+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:03.872+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:03.874+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:03.875+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:03.876+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:03.877+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:03.878+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:03.879+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:03.880+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:03.882+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:03.883+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:03.884+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:03.886+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:03.890+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:03.891+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:03.893+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:03.895+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:03.898+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:03.898+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:03.901+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:03.902+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:03.903+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:03.904+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:03.904+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:03.904+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:03.920+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:03.921+0000] {subprocess.py:93} INFO - 25/08/19 09:31:03 INFO BlockManagerInfo: Added broadcast_16_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:04.025+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 WARN TaskSetManager: Lost task 0.0 in stage 3.0 (TID 4) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:04.026+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:04.026+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:04.027+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:04.034+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:04.043+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:04.044+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:04.051+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:04.062+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:04.065+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:04.068+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:04.080+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:04.083+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:04.088+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:04.094+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:04.101+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:04.110+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:04.113+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:04.126+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:04.130+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:04.130+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:04.131+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:04.148+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:04.154+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:04.155+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:04.155+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:04.182+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:04.184+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:04.184+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 INFO BlockManagerInfo: Added broadcast_17_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:04.501+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 INFO TaskSetManager: Starting task 0.1 in stage 3.0 (TID 16) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:04.588+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 INFO TaskSetManager: Lost task 0.1 in stage 2.0 (TID 15) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:04.589+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:04.726+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 INFO BlockManagerInfo: Added broadcast_6_piece0 in memory on 172.18.0.16:37923 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:04.963+0000] {subprocess.py:93} INFO - 25/08/19 09:31:04 INFO TaskSetManager: Starting task 1.1 in stage 3.0 (TID 17) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:05.014+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 0.1 in stage 3.0 (TID 16) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:05.016+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:05.159+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO BlockManagerInfo: Added broadcast_8_piece0 in memory on 172.18.0.16:37923 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:05.227+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 0.2 in stage 3.0 (TID 18) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:05.240+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 WARN TaskSetManager: Lost task 1.0 in stage 6.0 (TID 13) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-51dc669b-3e8d-4eb1-a4ef-7ce39bbd2f31-c000.avro
[2025-08-19T09:31:05.241+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:05.242+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:05.245+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:05.246+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:05.247+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:05.248+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:05.248+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:31:05.248+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:05.249+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:05.259+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:05.259+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:05.264+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:05.265+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:05.265+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:05.266+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:05.269+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:05.270+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:05.271+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:05.272+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:05.273+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:05.273+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:05.273+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:05.274+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:05.276+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:05.277+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:05.283+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:05.283+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:05.342+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 0.2 in stage 2.0 (TID 19) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:05.360+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 WARN TaskSetManager: Lost task 0.0 in stage 6.0 (TID 12) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-51dc669b-3e8d-4eb1-a4ef-7ce39bbd2f31-c000.avro
[2025-08-19T09:31:05.361+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:05.362+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:05.365+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:05.366+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:05.368+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:05.373+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:05.376+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:31:05.377+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:05.379+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:05.382+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:05.389+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:05.389+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:05.390+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:05.390+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:05.391+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:05.391+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:05.401+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:05.402+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:05.403+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:05.403+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:05.412+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:05.413+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:05.414+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:05.418+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:05.419+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:05.421+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:05.422+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:05.523+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 1.1 in stage 2.0 (TID 20) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:05.543+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 1.1 in stage 3.0 (TID 17) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00011-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:05.544+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:05.625+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 1.2 in stage 3.0 (TID 21) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:05.647+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 WARN TaskSetManager: Lost task 0.0 in stage 7.0 (TID 14) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/US_category_id/part-00005-9eabfba6-2f34-4227-bfbd-ea61335f50f8-c000.avro
[2025-08-19T09:31:05.647+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:05.648+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:05.649+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:05.657+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:05.657+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:05.659+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:05.660+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage10.processNext(Unknown Source)
[2025-08-19T09:31:05.667+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:05.669+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:05.673+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:05.674+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:05.676+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:05.678+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:05.679+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:05.683+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:05.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:05.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:05.687+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:05.693+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:05.694+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:05.694+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:05.695+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:05.706+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:05.708+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:05.708+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:05.709+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:05.712+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:05.759+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 0.1 in stage 6.0 (TID 22) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:05.809+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 0.2 in stage 3.0 (TID 18) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:05.810+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:31:05.839+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 0.3 in stage 3.0 (TID 23) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:05.873+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 1.1 in stage 6.0 (TID 24) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:05.888+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 0.2 in stage 2.0 (TID 19) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:05.889+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:31:05.890+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 1.1 in stage 2.0 (TID 20) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:05.890+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:05.927+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Starting task 1.2 in stage 2.0 (TID 25) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:05.975+0000] {subprocess.py:93} INFO - 25/08/19 09:31:05 INFO TaskSetManager: Lost task 1.2 in stage 3.0 (TID 21) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00011-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:05.977+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:31:06.039+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 1.3 in stage 3.0 (TID 26) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:06.082+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Lost task 0.1 in stage 6.0 (TID 22) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-51dc669b-3e8d-4eb1-a4ef-7ce39bbd2f31-c000.avro
[2025-08-19T09:31:06.084+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:06.192+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 0.3 in stage 2.0 (TID 27) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:06.294+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Lost task 0.3 in stage 3.0 (TID 23) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:06.295+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:31:06.302+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 ERROR TaskSetManager: Task 0 in stage 3.0 failed 4 times; aborting job
[2025-08-19T09:31:06.328+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 0.2 in stage 6.0 (TID 28) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:06.413+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Lost task 1.2 in stage 2.0 (TID 25) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:06.415+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:31:06.492+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 1.3 in stage 2.0 (TID 29) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:06.531+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 0.1 in stage 7.0 (TID 30) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:06.553+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 1.0 in stage 7.0 (TID 31) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:06.657+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 0.0 in stage 8.0 (TID 32) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:06.710+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 1.0 in stage 8.0 (TID 33) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:06.714+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 0.0 in stage 9.0 (TID 34) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:06.821+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Starting task 1.0 in stage 9.0 (TID 35) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:06.833+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 INFO TaskSetManager: Lost task 1.1 in stage 6.0 (TID 24) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-51dc669b-3e8d-4eb1-a4ef-7ce39bbd2f31-c000.avro
[2025-08-19T09:31:06.834+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:06.840+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 WARN TaskSetManager: Lost task 0.0 in stage 0.0 (TID 0) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/RU_category_id/part-00005-1e73bc76-ce9f-45fa-9c14-9e6007d308ff-c000.avro
[2025-08-19T09:31:06.846+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:06.855+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:06.863+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:06.872+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:06.873+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:06.873+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:06.874+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage9.processNext(Unknown Source)
[2025-08-19T09:31:06.874+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:06.875+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:06.891+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:06.895+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:06.899+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:06.900+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:06.900+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:06.901+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:06.902+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:06.922+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:06.924+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:06.935+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:06.942+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:06.943+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:06.949+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:06.949+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:06.968+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:06.968+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:06.969+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:06.979+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:06.989+0000] {subprocess.py:93} INFO - 25/08/19 09:31:06 WARN TaskSetManager: Lost task 0.0 in stage 1.0 (TID 2) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/JP_category_id/part-00005-0eb0e845-cee5-420a-84bf-58f7e36e3e85-c000.avro
[2025-08-19T09:31:06.991+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:06.997+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:06.998+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:06.998+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:06.999+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.021+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.021+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage6.processNext(Unknown Source)
[2025-08-19T09:31:07.022+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.040+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.040+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.051+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:07.051+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:07.052+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:07.052+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:07.053+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:07.088+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:07.090+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:07.107+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:07.122+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:07.123+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:07.132+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:07.133+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:07.149+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:07.150+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:07.153+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:07.154+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:07.164+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:07.293+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Lost task 0.3 in stage 2.0 (TID 27) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:07.294+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:31:07.294+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 ERROR TaskSetManager: Task 0 in stage 2.0 failed 4 times; aborting job
[2025-08-19T09:31:07.295+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 WARN TaskSetManager: Lost task 1.0 in stage 1.0 (TID 3) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/JP_category_id/part-00011-0eb0e845-cee5-420a-84bf-58f7e36e3e85-c000.avro
[2025-08-19T09:31:07.300+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:07.306+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:07.317+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:07.318+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:07.331+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.332+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.332+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage6.processNext(Unknown Source)
[2025-08-19T09:31:07.332+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.333+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.333+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.333+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:07.334+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:07.362+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:07.364+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:07.364+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:07.365+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:07.381+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:07.393+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:07.393+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:07.406+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:07.406+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:07.427+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:07.428+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:07.439+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:07.440+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:07.444+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:07.444+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:07.458+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO BlockManagerInfo: Added broadcast_17_piece0 in memory on 172.18.0.12:37993 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:07.458+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Lost task 1.3 in stage 3.0 (TID 26) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00011-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:07.463+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:31:07.464+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Removed TaskSet 3.0, whose tasks have all completed, from pool
[2025-08-19T09:31:07.598+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Cancelling stage 3
[2025-08-19T09:31:07.602+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Killing all running tasks in stage 3: Stage cancelled: Job aborted due to stage failure: Task 0 in stage 3.0 failed 4 times, most recent failure: Lost task 0.3 in stage 3.0 (TID 23) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:07.603+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:07.603+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:07.603+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:07.615+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:07.618+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.635+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.637+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:07.637+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.639+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.645+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.646+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:07.646+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:07.647+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:07.647+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:07.665+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:07.666+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:07.667+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:07.679+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:07.679+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:07.700+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:07.700+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:07.701+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:07.702+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:07.710+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:07.713+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:07.714+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:07.714+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:07.715+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:31:07.729+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO BlockManagerInfo: Added broadcast_18_piece0 in memory on 172.18.0.12:37993 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:07.733+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 WARN TaskSetManager: Lost task 1.0 in stage 0.0 (TID 1) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/RU_category_id/part-00011-1e73bc76-ce9f-45fa-9c14-9e6007d308ff-c000.avro
[2025-08-19T09:31:07.740+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:07.741+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:07.746+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:07.747+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:07.747+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.755+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.755+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage9.processNext(Unknown Source)
[2025-08-19T09:31:07.763+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.763+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.768+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.771+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:07.771+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:07.772+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:07.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:07.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:07.787+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:07.787+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:07.792+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:07.793+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:07.794+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:07.804+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:07.809+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:07.809+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:07.810+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:07.810+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:07.818+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:07.820+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:07.826+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO DAGScheduler: ResultStage 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 229.067 s due to Job aborted due to stage failure: Task 0 in stage 3.0 failed 4 times, most recent failure: Lost task 0.3 in stage 3.0 (TID 23) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:07.827+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:07.828+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:07.831+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:07.836+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:07.837+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.838+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.839+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:07.850+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.851+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.851+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.852+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:07.852+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:07.852+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:07.870+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:07.870+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:07.871+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:07.878+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:07.887+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:07.888+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:07.889+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:07.889+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:07.890+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:07.890+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:07.891+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:07.904+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:07.905+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:07.905+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:07.906+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:31:07.922+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Starting task 1.1 in stage 0.0 (TID 36) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:07.930+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Lost task 0.2 in stage 6.0 (TID 28) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-51dc669b-3e8d-4eb1-a4ef-7ce39bbd2f31-c000.avro
[2025-08-19T09:31:07.936+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:31:07.937+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO BlockManagerInfo: Added broadcast_19_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:07.937+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Starting task 0.1 in stage 0.0 (TID 37) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:07.938+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO DAGScheduler: Job 2 failed: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264, took 229.641900 s
[2025-08-19T09:31:07.947+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSetManager: Lost task 1.3 in stage 2.0 (TID 29) on 172.18.0.16, executor 2: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:07.948+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:31:07.955+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO BlockManagerInfo: Added broadcast_19_piece0 in memory on 172.18.0.12:37993 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:07.964+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Removed TaskSet 2.0, whose tasks have all completed, from pool
[2025-08-19T09:31:07.983+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Cancelling stage 2
[2025-08-19T09:31:07.990+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO TaskSchedulerImpl: Killing all running tasks in stage 2: Stage cancelled: Job aborted due to stage failure: Task 0 in stage 2.0 failed 4 times, most recent failure: Lost task 0.3 in stage 2.0 (TID 27) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:07.991+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:07.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:07.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:07.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:07.995+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:07.995+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:07.995+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:31:07.996+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:07.996+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:07.997+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:07.999+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:08.000+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:08.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:08.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:08.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:08.003+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:08.004+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:08.005+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:08.005+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:08.006+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:08.009+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:08.009+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:08.011+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:08.013+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:08.014+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:08.015+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:08.015+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:08.016+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:31:08.018+0000] {subprocess.py:93} INFO - 25/08/19 09:31:07 INFO DAGScheduler: ResultStage 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 229.448 s due to Job aborted due to stage failure: Task 0 in stage 2.0 failed 4 times, most recent failure: Lost task 0.3 in stage 2.0 (TID 27) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-1803b48a-7bb6-4a9e-b8b4-9e32aa7c756e-c000.avro
[2025-08-19T09:31:08.018+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:08.019+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:08.022+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:08.022+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:08.025+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:08.026+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:08.026+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:31:08.027+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:08.028+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:08.031+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:08.032+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:08.033+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:08.034+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:08.035+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:08.037+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:08.037+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:08.038+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:08.038+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:08.039+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:08.041+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:08.043+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:08.043+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:08.044+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:08.045+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:08.047+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:08.049+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:08.050+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:08.051+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:31:08.060+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Job 3 failed: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264, took 229.790049 s
[2025-08-19T09:31:08.084+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSetManager: Starting task 1.1 in stage 1.0 (TID 38) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:08.110+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 WARN TaskSetManager: Lost task 1.0 in stage 7.0 (TID 31) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/US_category_id/part-00003-9eabfba6-2f34-4227-bfbd-ea61335f50f8-c000.avro
[2025-08-19T09:31:08.111+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:08.112+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:08.115+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:08.120+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:08.120+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:08.121+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:08.121+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage10.processNext(Unknown Source)
[2025-08-19T09:31:08.122+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:08.122+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:08.122+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:08.130+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:08.135+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:08.135+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:08.136+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:08.143+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:08.143+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:08.144+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:08.144+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:08.144+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:08.145+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:08.150+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:08.151+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:08.151+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:08.157+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:08.158+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:08.161+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:08.162+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:08.285+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId f0adbfa4-2ca7-490a-ae1b-ff3011688c76)
[2025-08-19T09:31:08.289+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 25c5af8d-8e6b-4380-9e25-8d8d95610baa)
[2025-08-19T09:31:08.377+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_10_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:08.378+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 71cf9c56-6d52-450e-8c63-b5423fac25ea)
[2025-08-19T09:31:08.379+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 4
[2025-08-19T09:31:08.390+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 5f68bbbe-b3fb-4011-b378-0c9ce317639b)
[2025-08-19T09:31:08.390+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 4: Stage cancelled: Job 4 cancelled part of cancelled job tag broadcast exchange (runId f0adbfa4-2ca7-490a-ae1b-ff3011688c76)
[2025-08-19T09:31:08.402+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 96f6e31b-4604-4299-bfda-098649656e12)
[2025-08-19T09:31:08.405+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 33a3bc72-f142-4915-8c99-094c543f21f2)
[2025-08-19T09:31:08.405+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 5c057802-e3bb-4a76-b083-63057089931a)
[2025-08-19T09:31:08.444+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_8_piece0 in memory on 172.18.0.12:37993 (size: 34.7 KiB, free: 434.3 MiB)
[2025-08-19T09:31:08.446+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_4_piece0 in memory on 172.18.0.12:37993 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:08.501+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId d21c10fe-2193-4afa-a055-479859d8697d)
[2025-08-19T09:31:08.518+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_11_piece0 in memory on 172.18.0.16:37923 (size: 6.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:08.703+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 4 was cancelled
[2025-08-19T09:31:08.705+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.066 s due to Job 4 cancelled part of cancelled job tag broadcast exchange (runId f0adbfa4-2ca7-490a-ae1b-ff3011688c76)
[2025-08-19T09:31:08.705+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 9
[2025-08-19T09:31:08.711+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 9: Stage cancelled: Job 9 cancelled part of cancelled job tag broadcast exchange (runId 25c5af8d-8e6b-4380-9e25-8d8d95610baa)
[2025-08-19T09:31:08.711+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 9 was cancelled
[2025-08-19T09:31:08.724+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 229.765 s due to Job 9 cancelled part of cancelled job tag broadcast exchange (runId 25c5af8d-8e6b-4380-9e25-8d8d95610baa)
[2025-08-19T09:31:08.728+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 8
[2025-08-19T09:31:08.730+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 8: Stage cancelled: Job 8 cancelled part of cancelled job tag broadcast exchange (runId 71cf9c56-6d52-450e-8c63-b5423fac25ea)
[2025-08-19T09:31:08.734+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 8 was cancelled
[2025-08-19T09:31:08.740+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 229.889 s due to Job 8 cancelled part of cancelled job tag broadcast exchange (runId 71cf9c56-6d52-450e-8c63-b5423fac25ea)
[2025-08-19T09:31:08.745+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 5
[2025-08-19T09:31:08.746+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 5: Stage cancelled: Job 5 cancelled part of cancelled job tag broadcast exchange (runId 5f68bbbe-b3fb-4011-b378-0c9ce317639b)
[2025-08-19T09:31:08.750+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 5 was cancelled
[2025-08-19T09:31:08.779+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.088 s due to Job 5 cancelled part of cancelled job tag broadcast exchange (runId 5f68bbbe-b3fb-4011-b378-0c9ce317639b)
[2025-08-19T09:31:08.789+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 1
[2025-08-19T09:31:08.790+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 1: Stage cancelled: Job 1 cancelled part of cancelled job tag broadcast exchange (runId 96f6e31b-4604-4299-bfda-098649656e12)
[2025-08-19T09:31:08.799+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 1 was cancelled
[2025-08-19T09:31:08.809+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.316 s due to Job 1 cancelled part of cancelled job tag broadcast exchange (runId 96f6e31b-4604-4299-bfda-098649656e12)
[2025-08-19T09:31:08.821+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_3_piece0 in memory on 172.18.0.12:37993 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:08.893+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Cancelling stage 6
[2025-08-19T09:31:08.895+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Killing all running tasks in stage 6: Stage cancelled: Job 6 cancelled part of cancelled job tag broadcast exchange (runId 33a3bc72-f142-4915-8c99-094c543f21f2)
[2025-08-19T09:31:08.905+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Removed TaskSet 6.0, whose tasks have all completed, from pool
[2025-08-19T09:31:08.920+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO TaskSchedulerImpl: Stage 6 was cancelled
[2025-08-19T09:31:08.924+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO DAGScheduler: ResultStage 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.201 s due to Job 6 cancelled part of cancelled job tag broadcast exchange (runId 33a3bc72-f142-4915-8c99-094c543f21f2)
[2025-08-19T09:31:09.065+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSetManager: Starting task 1.1 in stage 7.0 (TID 39) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:31:09.066+0000] {subprocess.py:93} INFO - 25/08/19 09:31:08 INFO BlockManagerInfo: Added broadcast_3_piece0 in memory on 172.18.0.16:37923 (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:09.086+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSetManager: Lost task 0.1 in stage 7.0 (TID 30) on 172.18.0.12, executor 0: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/US_category_id/part-00005-9eabfba6-2f34-4227-bfbd-ea61335f50f8-c000.avro
[2025-08-19T09:31:09.092+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:09.093+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Cancelling stage 0
[2025-08-19T09:31:09.093+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Killing all running tasks in stage 0: Stage cancelled: Job 0 cancelled part of cancelled job tag broadcast exchange (runId 5c057802-e3bb-4a76-b083-63057089931a)
[2025-08-19T09:31:09.094+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Stage 0 was cancelled
[2025-08-19T09:31:09.105+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO DAGScheduler: ResultStage 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.772 s due to Job 0 cancelled part of cancelled job tag broadcast exchange (runId 5c057802-e3bb-4a76-b083-63057089931a)
[2025-08-19T09:31:09.163+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSetManager: Starting task 0.2 in stage 7.0 (TID 40) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:31:09.247+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO BlockManagerInfo: Removed broadcast_3_piece0 on 172.18.0.16:37923 in memory (size: 34.7 KiB, free: 434.2 MiB)
[2025-08-19T09:31:09.256+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Cancelling stage 7
[2025-08-19T09:31:09.257+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Killing all running tasks in stage 7: Stage cancelled: Job 7 cancelled part of cancelled job tag broadcast exchange (runId d21c10fe-2193-4afa-a055-479859d8697d)
[2025-08-19T09:31:09.257+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Stage 7 was cancelled
[2025-08-19T09:31:09.258+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO DAGScheduler: ResultStage 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 230.469 s due to Job 7 cancelled part of cancelled job tag broadcast exchange (runId d21c10fe-2193-4afa-a055-479859d8697d)
[2025-08-19T09:31:09.281+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.0 in stage 8.0 (TID 32) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00005-791317cd-81d1-4a4b-add3-797769eb2832-c000.avro
[2025-08-19T09:31:09.293+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:09.298+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:09.298+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:09.307+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:09.308+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:09.320+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:09.321+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:31:09.330+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:09.330+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:09.331+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:09.331+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:09.347+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:09.348+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:09.348+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:09.349+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:09.350+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:09.356+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:09.357+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:09.357+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:09.357+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:09.358+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:09.369+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:09.370+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:09.370+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:09.370+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:09.371+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:09.377+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:09.378+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 8.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.394+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.0 in stage 8.0 (TID 33) (172.18.0.12 executor 0): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-791317cd-81d1-4a4b-add3-797769eb2832-c000.avro
[2025-08-19T09:31:09.401+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:09.406+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:09.418+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:09.420+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:09.421+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:09.432+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:09.432+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:31:09.432+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:09.433+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:09.443+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:09.446+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:09.447+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:09.448+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:09.465+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:09.466+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:09.467+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:09.468+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:09.475+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:09.476+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:09.481+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:09.491+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:09.492+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:09.500+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:09.504+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:09.506+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:09.514+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:09.515+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:09.519+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 8.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.553+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.1 in stage 1.0 (TID 38) (172.18.0.16 executor 2): TaskKilled (Stage cancelled: Job 1 cancelled part of cancelled job tag broadcast exchange (runId 96f6e31b-4604-4299-bfda-098649656e12))
[2025-08-19T09:31:09.686+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 1.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.700+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.1 in stage 0.0 (TID 37) (172.18.0.16 executor 2): TaskKilled (Stage cancelled: Job 0 cancelled part of cancelled job tag broadcast exchange (runId 5c057802-e3bb-4a76-b083-63057089931a))
[2025-08-19T09:31:09.709+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.0 in stage 9.0 (TID 34) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 9 cancelled part of cancelled job tag broadcast exchange (runId 25c5af8d-8e6b-4380-9e25-8d8d95610baa))
[2025-08-19T09:31:09.714+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSetManager: Lost task 1.1 in stage 7.0 (TID 39) on 172.18.0.12, executor 0: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/US_category_id/part-00003-9eabfba6-2f34-4227-bfbd-ea61335f50f8-c000.avro
[2025-08-19T09:31:09.719+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:31:09.730+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.1 in stage 0.0 (TID 36) (172.18.0.16 executor 2): TaskKilled (Stage cancelled: Job 0 cancelled part of cancelled job tag broadcast exchange (runId 5c057802-e3bb-4a76-b083-63057089931a))
[2025-08-19T09:31:09.732+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 0.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.784+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.0 in stage 9.0 (TID 35) (172.18.0.16 executor 2): TaskKilled (Stage cancelled: Job 9 cancelled part of cancelled job tag broadcast exchange (runId 25c5af8d-8e6b-4380-9e25-8d8d95610baa))
[2025-08-19T09:31:09.802+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 9.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.858+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.2 in stage 7.0 (TID 40) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 7 cancelled part of cancelled job tag broadcast exchange (runId d21c10fe-2193-4afa-a055-479859d8697d))
[2025-08-19T09:31:09.859+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 7.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.861+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.0 in stage 4.0 (TID 8) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 4 cancelled part of cancelled job tag broadcast exchange (runId f0adbfa4-2ca7-490a-ae1b-ff3011688c76))
[2025-08-19T09:31:09.907+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.0 in stage 5.0 (TID 11) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 5 cancelled part of cancelled job tag broadcast exchange (runId 5f68bbbe-b3fb-4011-b378-0c9ce317639b))
[2025-08-19T09:31:09.908+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 5.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.913+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 0.0 in stage 5.0 (TID 10) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 5 cancelled part of cancelled job tag broadcast exchange (runId 5f68bbbe-b3fb-4011-b378-0c9ce317639b))
[2025-08-19T09:31:09.959+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 5.0, whose tasks have all completed, from pool
[2025-08-19T09:31:09.975+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 WARN TaskSetManager: Lost task 1.0 in stage 4.0 (TID 9) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 4 cancelled part of cancelled job tag broadcast exchange (runId f0adbfa4-2ca7-490a-ae1b-ff3011688c76))
[2025-08-19T09:31:09.976+0000] {subprocess.py:93} INFO - 25/08/19 09:31:09 INFO TaskSchedulerImpl: Removed TaskSet 4.0, whose tasks have all completed, from pool
[2025-08-19T09:31:13.973+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-08-19T09:31:13.974+0000] {subprocess.py:93} INFO -   File "/opt/***/jobs/transformation/transform_to_golden_dataset.py", line 135, in <module>
[2025-08-19T09:31:13.974+0000] {subprocess.py:93} INFO -     final_df.write.format("parquet").mode("overwrite").save("hdfs://namenode:9000/storage/hdfs/processed/golden_dataset")
[2025-08-19T09:31:13.975+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 1463, in save
[2025-08-19T09:31:13.975+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2025-08-19T09:31:13.975+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2025-08-19T09:31:13.976+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2025-08-19T09:31:13.979+0000] {subprocess.py:93} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o1969.save.
[2025-08-19T09:31:13.980+0000] {subprocess.py:93} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 3.0 failed 4 times, most recent failure: Lost task 0.3 in stage 3.0 (TID 23) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:13.981+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:13.981+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:13.985+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:13.985+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:13.986+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:13.986+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:13.987+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:13.987+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:13.988+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:13.988+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:13.988+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:13.989+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:13.989+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:13.989+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:13.991+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:13.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:13.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:13.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:13.992+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:13.993+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:13.993+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:13.993+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:13.994+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:13.994+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:13.995+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:13.995+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:13.995+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:13.996+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:31:13.996+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2025-08-19T09:31:13.996+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2025-08-19T09:31:13.997+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2025-08-19T09:31:13.997+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2025-08-19T09:31:13.997+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2025-08-19T09:31:13.998+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2025-08-19T09:31:13.998+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2025-08-19T09:31:13.998+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2025-08-19T09:31:13.999+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2025-08-19T09:31:13.999+0000] {subprocess.py:93} INFO - 	at scala.Option.foreach(Option.scala:407)
[2025-08-19T09:31:13.999+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2025-08-19T09:31:14.000+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2025-08-19T09:31:14.000+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2025-08-19T09:31:14.000+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2025-08-19T09:31:14.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2025-08-19T09:31:14.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2025-08-19T09:31:14.001+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2398)
[2025-08-19T09:31:14.002+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2419)
[2025-08-19T09:31:14.002+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2438)
[2025-08-19T09:31:14.002+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2463)
[2025-08-19T09:31:14.003+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$collect$1(RDD.scala:1049)
[2025-08-19T09:31:14.003+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
[2025-08-19T09:31:14.003+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
[2025-08-19T09:31:14.004+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.withScope(RDD.scala:410)
[2025-08-19T09:31:14.004+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.collect(RDD.scala:1048)
[2025-08-19T09:31:14.005+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.executeCollectIterator(SparkPlan.scala:455)
[2025-08-19T09:31:14.005+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.exchange.BroadcastExchangeExec.$anonfun$relationFuture$1(BroadcastExchangeExec.scala:140)
[2025-08-19T09:31:14.006+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withThreadLocalCaptured$2(SQLExecution.scala:224)
[2025-08-19T09:31:14.006+0000] {subprocess.py:93} INFO - 	at org.apache.spark.JobArtifactSet$.withActiveJobArtifactState(JobArtifactSet.scala:94)
[2025-08-19T09:31:14.007+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withThreadLocalCaptured$1(SQLExecution.scala:219)
[2025-08-19T09:31:14.007+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2025-08-19T09:31:14.009+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:31:14.010+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:31:14.010+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:31:14.011+0000] {subprocess.py:93} INFO - Caused by: org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/GB_category_id/part-00005-b84e8589-5344-4af6-803c-c6710e944bb2-c000.avro
[2025-08-19T09:31:14.012+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:31:14.012+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:31:14.013+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:31:14.014+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:31:14.014+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:31:14.015+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:31:14.015+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage4.processNext(Unknown Source)
[2025-08-19T09:31:14.016+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:31:14.016+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:31:14.016+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:31:14.017+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:31:14.017+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:31:14.017+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:31:14.018+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:31:14.018+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:31:14.018+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:31:14.019+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:31:14.019+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:31:14.019+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:31:14.020+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:31:14.020+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:31:14.020+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:31:14.021+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:31:14.021+0000] {subprocess.py:93} INFO - 	... 3 more
[2025-08-19T09:31:14.021+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:31:14.807+0000] {subprocess.py:93} INFO - 25/08/19 09:31:14 INFO SparkContext: Invoking stop() from shutdown hook
[2025-08-19T09:31:14.812+0000] {subprocess.py:93} INFO - 25/08/19 09:31:14 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-08-19T09:31:14.844+0000] {subprocess.py:93} INFO - 25/08/19 09:31:14 INFO SparkUI: Stopped Spark web UI at http://b208641feb52:4043
[2025-08-19T09:31:14.866+0000] {subprocess.py:93} INFO - 25/08/19 09:31:14 INFO StandaloneSchedulerBackend: Shutting down all executors
[2025-08-19T09:31:14.868+0000] {subprocess.py:93} INFO - 25/08/19 09:31:14 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2025-08-19T09:31:16.159+0000] {subprocess.py:93} INFO - 25/08/19 09:31:16 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-08-19T09:31:17.090+0000] {subprocess.py:93} INFO - 25/08/19 09:31:17 INFO MemoryStore: MemoryStore cleared
[2025-08-19T09:31:17.090+0000] {subprocess.py:93} INFO - 25/08/19 09:31:17 INFO BlockManager: BlockManager stopped
[2025-08-19T09:31:17.505+0000] {subprocess.py:93} INFO - 25/08/19 09:31:17 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-08-19T09:31:17.828+0000] {subprocess.py:93} INFO - 25/08/19 09:31:17 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-08-19T09:31:18.674+0000] {subprocess.py:93} INFO - 25/08/19 09:31:18 INFO SparkContext: Successfully stopped SparkContext
[2025-08-19T09:31:18.705+0000] {subprocess.py:93} INFO - 25/08/19 09:31:18 INFO ShutdownHookManager: Shutdown hook called
[2025-08-19T09:31:18.835+0000] {subprocess.py:93} INFO - 25/08/19 09:31:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-671c8c7d-e1dc-45f7-8fd5-5da643f4bf8e
[2025-08-19T09:31:18.909+0000] {subprocess.py:93} INFO - 25/08/19 09:31:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-671c8c7d-e1dc-45f7-8fd5-5da643f4bf8e/pyspark-645acc39-0032-453c-9019-747c9b98db16
[2025-08-19T09:31:18.960+0000] {subprocess.py:93} INFO - 25/08/19 09:31:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-3a22e7a7-b4b7-4cb5-8d79-dd233661fe2d
[2025-08-19T09:31:19.910+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-19T09:31:19.915+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-19T09:31:20.070+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-19T09:31:20.142+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_pipeline_batch_data, task_id=transform_golden, execution_date=20250819T091131, start_date=20250819T092518, end_date=20250819T093120
[2025-08-19T09:31:21.251+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 606 for task transform_golden (Bash command failed. The command returned a non-zero exit code 1.; 2392)
[2025-08-19T09:31:21.322+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-19T09:31:21.554+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-19T09:31:21.560+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
